ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -8,15 +8,15 @@
 
 | Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | Channel-Group |
 | --------- | ----------- | ---- | ----- | ----------- | ----------- | ------------- |
-{%     for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
-{%         if ethernet_interfaces[ethernet_interface].channel_group.id is arista.avd.defined %}
-{%             set port_channel_interface = 'Port-Channel' + ethernet_interfaces[ethernet_interface].channel_group.id | string %}
+{%     for ethernet_interface in ethernet_interfaces | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+{%         if ethernet_interface.channel_group.id is arista.avd.defined %}
+{%             set port_channel_interface = 'Port-Channel' + ethernet_interface.channel_group.id | string %}
 {%             if port_channel_interfaces[port_channel_interface].type is not defined or port_channel_interfaces[port_channel_interface].type not in ['routed', 'l3dot1q', 'l2dot1q'] %}
-{%                 set description = ethernet_interfaces[ethernet_interface].description | default("-") %}
+{%                 set description = ethernet_interface.description | default("-") %}
```

Comment:
Another change we needed. Replace **default** filter with **arista.avd.default**